<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lobby</class>
 <widget class="QWidget" name="Lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_salon">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>630</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Merriweather Black</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salon</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_joupre">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>630</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Merriweather Black</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Joueur(s) pr√©sent(s) :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_joueur4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>611</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Merriweather Black</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_joueur7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>680</y>
     <width>611</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Merriweather Black</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>416</y>
     <width>431</width>
     <height>541</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>600</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="message5">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>630</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="message8">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>746</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_envoi">
   <property name="geometry">
    <rect>
     <x>1690</x>
     <y>866</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="message3">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>560</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="envoi_msg">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>866</y>
     <width>321</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="message4">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>600</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="message6">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>670</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="message0">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>429</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="message10">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>826</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="message2">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>520</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="message9">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>786</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="message1">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>470</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="message7">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>710</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_retour">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>350</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Merriweather</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>320</y>
     <width>160</width>
     <height>156</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="playerList"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>210</y>
     <width>431</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="canstartlayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
